package.json

```diff
@@ -3,14 +3,17 @@
     "private": true,
     "type": "module",
     "scripts": {
-        "dev": "concurrently 'pnpm run watch:content' 'pnpm run dev:docs'",
-        "dev:docs": "pnpm --stream --filter {docs} dev:docs",
-        "watch:content": "chokidar 'packages/components/**/*.mdx' -c 'pnpm run dev:content'",
-        "dev:content": "pnpm --stream --filter {docs} dev:contentlayer",
+        "storybook": "concurrently 'pnpm run storybook:new' 'pnpm run storybook:legacy' 'pnpm run storybook:main'",
+        "storybook:new": "pnpm --stream  --filter {packages/components} storybook:background",
+        "storybook:legacy": "pnpm --stream  --filter {packages/fondue} storybook:background",
```

Comment:
do we need two separate scripts? What we want to do is to just run all available background instances.

We can do that with `pnpm -r storybook:watch`.

Reply:
I replaced them with the following script

`"storybook:packages": "pnpm -parallel --filter {packages/*} storybook:background",`

Reply:
I think we can improve it further by doing `"pnpm -parallel --filter {packages/*} storybook -- --no-open"`

Reply:
good catch! i adjusted the script

`"storybook:packages": "pnpm --parallel --filter {packages/*} storybook --no-open"`